mindsdb/integrations/handlers/shopify_handler/README.md

```diff
@@ -118,3 +118,14 @@ LIMIT 5
 ~~~~
 
 `inventory_item_ids` or `location_ids` have to be specified in the `where` clause of the query. 
+
+`location_ids` can be queried as follows:
+
+~~~~sql
+SELECT id as location_id, name, address1, city, zip 
```

Comment:
The alias `id as location_id` doesn't work - the output column is `id` instead of `location_id`.